src/map/pc.hpp

```diff
@@ -63,6 +63,8 @@ enum sc_type : int16;
 #define ATTENDANCE_DATE_VAR "#AttendanceDate"
 #define ATTENDANCE_COUNT_VAR "#AttendanceCounter"
 #define ACHIEVEMENTLEVEL "AchievementLevel"
+#define GOLDPC_POINT_VAR "Goldpc_Points"
```

Comment:
Are the Points per Character or Per Account?

```
#define GOLDPC_POINT_VAR "#Goldpc_Points"
#define GOLDPC_SECONDS_VAR "#Goldpc_Seconds"

```

Reply:
i think per Character.

Reply:
if it is by character it must change:

`sd->goldpcPoints = static_cast<int>(pc_readaccountreg(sd, add_str(GOLDPC_POINT_VAR)));
`
suggested in my review to:

`sd->goldpcPoints = static_cast<int>(pc_readglobalreg(sd, add_str(GOLDPC_POINT_VAR)));`

Reply:
I have tested it and it solves the Client-Server synchronization problem and the problem described by @ecdarreola

Reply:
sd->goldpcPoints = static_cast<int>(pc_readaccountreg(sd, add_str(GOLDPC_POINT_VAR)));

It good working.

Reply:
just remember to add goldpcPoins in pc.hpp next to int cashPoints;

Reply:
there is a conflict in feature.conf
![image](https://user-images.githubusercontent.com/1854368/202090066-980261b6-b92c-40e8-897a-c98cf31c8d2d.png)

Reply:
Are you comparing two different branches?
If you want to implement this, try it manually.

Reply:
im just applying the diff patch to the master and i just got that error. and also just got this error message on the map.
[Error]: clif_parse_dynamic_npc: Original NPC "GOLDPCCAFE" was not found.

Reply:
Could you please keep support requests to Discord?
Like @Tolimatoi said you merged it wrong yourself and yes it has a conflict and you need to manually resolve it for the time being. All other issues are most likely follow up issues because you relied on something that did not work automatically like you expected.